mlkem/poly.c

```diff
@@ -612,6 +614,7 @@ void poly_tomont(poly *r)
 *
 * Arguments:   - poly *r: pointer to input/output polynomial
 **************************************************/
+#if !defined(MLKEM_USE_AARCH64_ASM)
```

Comment:
Please move this pre-processor conditional inside of `poly_reduce` - just like for `ntt()`